torch/onnx/symbolic.py

```diff
@@ -1488,3 +1488,31 @@ def log_sigmoid(g, input):
 @parse_args('v')
 def erf(g, input):
     return g.op('Erf', input)
+
+
+@parse_args('v', 'i', 'i')
+def flatten(g, input, start_dim, end_dim) :
+    input_dims = input.type().sizes()
```

Comment:
Here we should use `dim` to access the dimension information. And after you get the `dim`, please check whether it's `CompleteTensorType` or not, to call `sizes`.

More details, please check https://github.com/pytorch/pytorch/commit/1235aa4fca55cea7c87febcc2761b84bf3114f1a#diff-9488e7f53a53fa19bef4dbae6a0b093a